src/flows/shared/steps/OrganizeCollection/Editor/StagingArea.tsx

```diff
@@ -66,6 +63,9 @@ function StagingArea() {
       >
         <SortableContext items={stagedNfts}>
           <StyledStagedNftContainer>
+            <StyledHeadingWrapper>
+              <Heading>Your collection</Heading>
+            </StyledHeadingWrapper>
```

Comment:
oo if we move the heading into the DND, we could remove the fixed width from the heading, and set it to width: 100% instead
on line 89